slick/slick.js

```diff
@@ -1358,23 +1358,26 @@ Issues: http://github.com/kenwheeler/slick/issues
 
 	Slick.prototype.initDotEvents = function() {
 
-		var _ = this;
-
-		if (_.options.dots === true && _.slideCount > _.options.slidesToShow) {
-			$('li', _.$dots).on('click.slick', {
```

Comment:
please see https://api.jquery.com/jquery/#selector-context about `$('li', _.$dots)`

in short, the equivalent to this is `_.queryAll('li', _.$dots.get(0))` and not `_.$dots.get(0).children`
https://github.com/savoirfairelinux/vanilla-slick/blob/develop/slick/slick.js#L3001